requirements.txt

```diff
@@ -1,5 +1,5 @@
 jinja2
-kubernetes ~= 12.0
+kubernetes >= 12.0
```

Comment:
This has gotten quite far behind, but I'm wary to automatically upgrade X versions. WDYT about `kubernetes ~= 21.7.0`

It looks like they bump the X quite frequently though

Reply:
That’s why I use >=, when this PR introduced on 2021 Nov it was still 19.x, and now it is already 21.x.

Using ~= is basically a good idea, but seems not for case in here :-(

Reply:
From the other side, I may able to explain why python-kubernetes update its x. version but not x.y.
* Kubernetes API coming with a N/N-1 support policy for many years
* In case python-kubernetes need to sync its support with Kubernetes corresponding API version AND avoid end user incorrectly install a non-compatible version, they bump the x.
* In case we also hope to sync with python-kubernetes handling, then we should using both >= and < in our requirements AND/OR also having a x. release bump with similar testing policy